README.md

```diff
@@ -21,11 +29,16 @@ Navigate to the **Config** tab of your robot’s page in [the Viam app](https://
 > [!NOTE]  
 > For more information, see [Configure a Robot](https://docs.viam.com/manage/configuration/).
 
+To save your changes, click **Save config** at the bottom of the page.
+Check the **Logs** tab of your robot in the Viam app to make sure your RPlidar has connected and no errors are being raised.
+
 ### Attributes
 
 No attributes are available for a `lidar:rplidar` camera.
 
-## Local installation
+## Build and Run locally
+
+If you don't want to load the model from the registry, you can install it locally. Follow these instructions to [configure a local module on your machine](https://docs.viam.com/registry/configure/#edit-the-configuration-of-a-local-module).
```

Comment:
```suggestion
If you don't want to load the model from the registry, for example because you are actively changing its functionality, you can install it locally. Follow these instructions to [configure a local module on your machine](https://docs.viam.com/registry/configure/#edit-the-configuration-of-a-local-module).
```